src/commands/install.yml

```diff
@@ -4,11 +4,10 @@ parameters:
     description: Where to install the CLI.
     type: string
     default: /usr/local/bin
-  channel:
```

Comment:
Can/should we leave this argument, mark it as deprecated, and change the implementation to complain loudly/tell someone what they need to do to fix it? That way the upgrade path is a bit easier for anyone using this.